### PR TITLE
Feature/shot mask

### PR DESCRIPTION
**Pull request by chriskalmar**
Proof of concept for masking page elements using CSS selectors.
Based on request in #186

### CODE REVIEW

File: src/shots/shots.ts [hunk 1]
```diff
@@ -15,9 +15,10 @@ const takeScreenShot = async ({
   browser: Browser;
   shotItem: ShotItem;
   logger: (message: string, ...rest: unknown[]) => void;
-}) => {
+}): Promise<boolean> => {
   const context = await browser.newContext(shotItem.browserConfig);
   const page = await context.newPage();
+  let success = false;
```
Comment (d-ivashchuk) [inline src/shots/shots.ts:21]: Why do we need this?

File: src/crawler/storybook.ts [hunk 2]
```diff
@@ -265,6 +267,7 @@ const generateBrowserConfig = (story: Story) => {
 export const generateStorybookShotItems = (
   baseUrl: string,
   stories: Story[],
+  mask?: Mask[],
```
Comment (d-ivashchuk) [inline src/crawler/storybook.ts:270]: Where is the actual css that masks the stuff, am I blind?

**Review by d-ivashchuk — APPROVED** [review]

File: src/shots/shots.ts (hunk 1 above)
Comment (chriskalmar) [reply src/shots/shots.ts:21]: A bit lower it is set to `true` if the screenshot could be taken successfully.
I introduced this so we can display a different message if the shot didn't happen.

File: src/crawler/storybook.ts (hunk 2 above)
Comment (chriskalmar) [reply src/crawler/storybook.ts:270]: You define here the list of CSS selectors:
https://github.com/lost-pixel/lost-pixel/blob/feature/shot-mask/src/config.ts#L173-L186

**Pull request merged**